lib/v2/cztv/zjxwlb.js

```diff
@@ -0,0 +1,34 @@
+const cheerio = require('cheerio');
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+module.exports = async (ctx) => {
+    const url = 'http://www.cztv.com/videos/zjxwlb';
+
+    const { data: res } = await got(url);
+    const $ = cheerio.load(res);
+    const list = $('#videolistss li')
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            return {
+                title: item.find('span.t1').text(),
+                link: item.find('input[name=data-url]').attr('value'),
+                pubDate: timezone(parseDate(item.find('span.t2').text() + ' 16:30', 'YYYY-MM-DD hh:mm'), +8),
+            };
+        });
+
+    const out = {
+        title: list[0].title,
+        link: list[0].link,
+        pubDate: list[0].pubDate,
+        description: list.map((item) => `<a href="${item.link}">${item.title}</a>`).join('<br>'),
```

Comment:
The site also provides a direct URL for the video and a poster. Maybe you can use that instead of a link.

Reply:
That's the reason why `item.title` exist. `title` is supposed to tell users briefly about a feed item. Moreover, you can present the video in `description` instead of asking users to visit an URL.